canonical/design.ubuntu.com
# Fix favicon not appearing in Google search results

## Done

Similar to https://github.com/canonical/vanilla-framework/pull/5183, this updates the favicon used by this site to a 48x48 size so that it can be rendered in Google search results.

This also updates the favicon from [old circle design](https://assets.ubuntu.com/v1/49a1a858-favicon-32x32.png) to [new square design](https://assets.ubuntu.com/v1/be7e4cc6-COF-favicon-32x32.png).

Copies [favicon configuration from c.com](https://github.com/canonical/canonical.com/blob/f866a0da17f41e9e02f8a6f6786483ea040f5ce9/templates/base_index.html#L87-L104).

## Google Recrawl
After this is merged, it may take a few days for the icon to change on Google search results. If it still hasn't been fixed after some time, we should [submit a recrawl request](https://developers.google.com/search/docs/crawling-indexing/ask-google-to-recrawl) which should fix the issue.

## QA
- Open [demo](https://design-ubuntu-com-319.demos.haus/).
- Verify tab favicon renders as expected.
- Open [locally served favicon](https://design-ubuntu-com-319.demos.haus/static/images/favicons/COF-favicon-48x48.png).
- Verify the favicon renders as expected.

## Issue / Card
Fixes [WD-12711](https://warthogs.atlassian.net/browse/WD-12711)

[WD-12711]: https://warthogs.atlassian.net/browse/WD-12711?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
